Sources/BraveShared/Resources/de.lproj/BraveShared.strings

```diff
@@ -1288,9 +1288,6 @@
 /* The description indicating when is the open tab synced. The parameter substituted for \"%@\" is the actual string representation of the exact time. E.g.: Yesterday 12:00 PM */
 "opentabs.openTabsItemLastSyncedYesterdayTitle" = "Gestern um %@";
 
-/* The title displayed on table header for open tabs list from other devices */
-"openTabs.openTabsListTableHeaderTitle" = "Geräte";
```

Comment:
I missed two entries, they are updated in transifex and will be updated next time we pull